6001-7000/LIT6018MalkaSanbat.xml

```diff
@@ -0,0 +1,117 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ሰንበተ፡ ክርስቲያን፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Malkǝʾa sanbata krǝstiyān</title>
+                <title xml:lang="en" corresp="#t1">Image of the Sabbath of the Christians</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="ES"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="GundaGunde144"></witness>
+                    <witness corresp="BAVet276"></witness>
+                    <witness corresp="EMML6993"></witness>
+                    <witness corresp="EAP526112"></witness>
+                </listWit>
+                
+                
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change when="2024-08-14" who="ES">created record as suggested in issue https://github.com/BetaMasaheft/Documentation/issues/2604 by Augustine Dickinson</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listRelation><relation name="saws:isDifferentTo" active="LIT6018MalkaSanbat" passive="LIT2889RepCh170"></relation>
+                <relation name="saws:isDifferentTo" active="LIT6018MalkaSanbat" passive="LIT6179MalkaSanbat"></relation></listRelation>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart">
+                    <lg type="stanza" n="1">
+                        <l n="1">ድርሳን፡ ዘደረሰ፡ አሐዱ፡ ርቱዐ፡ ሃይማኖት፨</l>
```

Comment:
Please note that this poem does not appear like malke', even though it may be called like this, somewhere. Is there by chance any other title?

Reply:
I followed what proposed by @smaugustine https://github.com/BetaMasaheft/Documentation/issues/2604

Reply:
It may not be a malkəʾ in the absolute strictest sense, as it is a poem that does not salute any body parts, but it is firmly a part of the malkəʾ tradition and is titled as such consistently in manuscripts. The same is true for many other texts, like Malkəʾa Fəlsatā and Malkəʾa Śəʿl and Malkəʾa Ləssān, even versions of Malkəʾa ʾƎgziʾabəḥer ʾAb, and these are nonetheless consistently titled as malkəʾ both in manuscripts containing them, in manuscripts referencing them, and in modern printed copies and texts prepared by marigetas.

In this particular case, its title is also confirmed by the list of titles present in several manuscripts (which I have presented on previously).

![image](https://github.com/user-attachments/assets/73c681a6-91da-4285-8c46-6eb6070107a9)
<img width="1152" alt="image" src="https://github.com/user-attachments/assets/aadd06c8-6fa0-466f-974a-4f9dfb666c83">
![image](https://github.com/user-attachments/assets/86ad439f-e2ce-4d00-b29a-0531bc00de16)

Reply:
I should add that, while not clear from the stanzas presented here (mea culpa), most of the text has "ሰላም፡ ለ-" stanzas, though to things other than body parts apparently since this text, unlike others, avoids assigning body parts to a non-person.

Reply:
I see. I would just remark that the tradition and marigetas have their own views and attitudes which sometimes collide with the western approaches, in this case of the idea of text genres ...  I wonder if it makes sense to add in such cases a supplied English title, as t2 or t3 ect., which would give a better hint to the form and genre (smth like "Poetic praise to ... alluding ..." - whatever). The traditional title may stay as it is. A short explanation in abstract might be also helpful.  Otherwise the user may be for a short while confused. Malke' poetry is actually well known in Ethiopicist circles and beyond, and what it is about. An English supplied title would better actually show that at a certain point the rigit formal framework of the malke'- genre started to be transgressed.

Reply:
This seems fair. For my part, I distinguish malkəʾ (saluting body parts), pseudo-malkəʾ (saluting other things in sequence), and related poetry (sharing the same poetic structure and found together) while accepting that these are my own classifications to facilitate the study of the genre in the broad sense presented in manuscripts.

On the last point, I would suggest, considering the extant evidence, that it is rather the opposite, though: that it was not initially a rigid framework that was later transgressed but that it was a rather loose framework which became rigid after certain texts were canonized within the genre (i.e. during the Gondarine period, when the now-standard መልክአ፡ ኢየሱስ፡, መልክአ፡ ማርያም፡, and መልክአ፡ ሥላሴ፡ were composed).